ruma-federation-api/src/discovery/get_server_keys/v2.rs

```diff
@@ -1,9 +1,7 @@
 //! [GET /_matrix/key/v2/server](https://matrix.org/docs/spec/server_server/r0.1.3#get-matrix-key-v2-server-keyid)
```

Comment:
Can you update this to use the r0.1.4 spec?

```suggestion
//! [GET /_matrix/key/v2/server](https://matrix.org/docs/spec/server_server/r0.1.4#get-matrix-key-v2-server-keyid)
```